src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -260,7 +260,7 @@ private Card createJobCompletedCard() {
 
         addCauses(factsList);
 
-        String activityTitle = "Update from build " + jobName + ".";
+        String activityTitle = "Update from " + jobName + ".";
```

Comment:
I still think the title could be better for completed, but feedback would be appreciated